### PR TITLE
DRILL-7592: Add missing licenses and update plugins exclusion list

### PR DESCRIPTION
**Pull request by vvysotskyi**
# [DRILL-7592](https://issues.apache.org/jira/browse/DRILL-7592): Add missing licenses and update plugins exclusion list

## Description

Updated list of files to check, updated their license.

## Documentation
NA

## Testing
Checked that both rat and license-maven-plugin checks are passed.

### CODE REVIEW

File: common/src/main/resources/drill-module.conf [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 
 //  This file tells Drill to consider this module when class path scanning.
 //  This file can also include any supplementary configuration information.
```
Comment (agozhiy) [inline common/src/main/resources/drill-module.conf]: Please bring the comments to the same style here and in the other places ("//" -> "#").

**Comment by agozhiy** [comment]
I wonder if there some way to add licence headers to generated files at contrib/native/client/src/protobuf?

File: common/src/main/resources/drill-module.conf (hunk 1 above)
Comment (vvysotskyi) [reply common/src/main/resources/drill-module.conf]: Thanks, fixed all the comments.

**Comment by agozhiy** [comment]
+1. Thank you for the changes. Please, squash the commits.

**Pull request closed without merging**